lib/uo_cli.rb

```diff
@@ -74,7 +74,7 @@ def initialize
       # Define creation commands
       def opt_create
         @subopts = Optimist.options do
-          banner "\nURBANopt #{@command}:\n \n"
+          banner "\nURBANopt create:\n \n"
```

Comment:
Not sure why `command` isn't being parsed in the banner, but it was just blank. By using a string primitive we can at least show the user where they are.